### ISSUE TITLE
Accessibility issue related to form element labels

### ISSUE THREAD

**Issue by joelhsmith** [smashingmagazine/redesign#144]
**- Do you want to request a *feature* or report a *bug*?**

Bug report

**- What is the current behavior?**
Forms elements missing text in labels

**- If the current behavior is a bug, please provide the steps to reproduce.**

Some form elements need labels with text in them.  
1x Missing form label
1x Empty form label

Easiest way to reproduce/ see is viewing the WAVE report. 
http://wave.webaim.org/report#/https://next.smashingmagazine.com/2017/03/a-little-surprise-is-waiting-for-you-here--meet-the-next-smashing-magazine/

**- What is the expected behavior?**
Labels should have text in them that describes what they label.

**- Please mention your operating system version and the version of your browser**
Using OSX Chrome and WAVE Toolbar

**Comment by vitalyfriedman** [comment]
Thank you very much for your feedback, @joelhsmith — adding on our to-do list!
